packages/backend/src/controllers/house/house.controller.ts

```diff
@@ -73,7 +73,7 @@ export class HouseController {
     @User() user: DecodedIdToken,
   ): Promise<HouseResponseDto | null> {
     const userDoc = await this.userStoreService.findOneByFirebaseId(user.uid);
-    if (userDoc.house != undefined) {
+    if (userDoc?.house != undefined) {
```

Comment:
should we consider using strict equality operator, i.e !==

Reply:
Not sure I just left it the same. I imagine it's just to cover checking both bull and undefined.

Reply:
yes that is why it's there